doc/sphinx/language/core/basic.rst

```diff
@@ -118,8 +118,10 @@ Numbers
   a number without fractional nor exponent parts. :n:`@nat` is a non-negative
   integer.  Underscores embedded in the digits are ignored, for example
   ``1_000_000`` is the same as ``1000000``.
+  :n:`@int63` (resp. :n:`@nat63`) are :n:`@int` (resp. :n:`@nat`) that fit
+  into an OCaml integer (i.e., 63bit integers on most platforms).
```

Comment:
```suggestion
  :n:`@int63` and :n:`@nat63` represent integers and natural numbers that fit
  into an OCaml integer (i.e., 63-bit integers on most platforms).
```

I suppose most people are using 63-bit integers these days and that 31-bit integers are passé. 

Using "resp." is bit pedantic.  I doubt anyone would misinterpret this simpler language.

Reply:
> I suppose most people are using 63-bit integers these days and that 31-bit integers are passé.

I would say yes though we still maintain compatibility of Coq with 32-bit systems.
Note also that in JSCoq (https://github.com/jscoq/jscoq) we have 32-bit integers (not 31, don't ask me the details).